Makefile.core.mk

```diff
@@ -336,9 +336,7 @@ lint-helm-global:
 	find manifests -name 'Chart.yaml' -print0 | ${XARGS} -L 1 dirname | xargs -r helm lint --strict -f manifests/global.yaml
 
 lint: lint-python lint-copyright-banner lint-scripts lint-go lint-dockerfiles lint-markdown lint-yaml lint-licenses lint-helm-global
-	@bin/check_helm.sh
```

Comment:
Same - we should still run a check_helm, but with the new templates.

Can be separate PR.

Reply:
We do the same as `lint-helm-global`